workflow/snakemake_rules/nextclade.smk

```diff
@@ -84,7 +84,7 @@ checkpoint get_sequences_without_nextclade_annotations:
                 --input_tsv={input.nextclade} \
                 --output_fasta={output.fasta}
         else
-            cp {input.fasta} {output.fasta}
```

Comment:
I'm not sure this `mv` is safe to do considering the `input.fasta` (`f"data/{database}/sequences.fasta"`) is required for upload. I don't think there's anything in place to guarantee that the upload of `sequences.fasta` occurs before this rule runs?